src/interfaces/swig/ParameterObserver.i

```diff
@@ -14,6 +14,7 @@
 
 namespace shogun {
 
+%template(SomeObservedValue) Some<ObservedValue>;
 %template(get_CV_storage) CSGObject::get<CrossValidationStorage*, void>;
```

Comment:
Since I need to expose Some<ObservedValue> to SWIG, I've tried to add it to the build. However, I keep getting these warnings:
```
/root/shogun/src/shogun/base/some.h:71: Warning 508: Declaration of 'get' shadows declaration accessible via operator->(),              
/root/shogun/src/shogun/base/SGObject.h:487: Warning 508: previous declaration of 'get'.                                                        
/root/shogun/src/shogun/base/some.h:71: Warning 508: Declaration of 'get' shadows declaration accessible via operator->(),                      
/root/shogun/src/shogun/base/SGObject.h:496: Warning 508: previous declaration of 'get'.                                                        
/root/shogun/src/shogun/base/some.h:24: Warning 509: Overloaded method shogun::Some< shogun::ObservedValue >::Some(shogun::Some< shogun::Observe
dValue > &&) effectively ignored,                                                                                                               
/root/shogun/src/shogun/base/some.h:20: Warning 509: as it is shadowed by shogun::Some< shogun::ObservedValue >::Some(shogun::Some< shogun::Obse
rvedValue > const &).                                                                                                                           
/root/shogun/src/shogun/base/some.h:46: Warning 503: Can't wrap 'operator shogun::ObservedValue*' unless renamed to a valid identifier. 
```
Moreover, the entire compilations fails with this obscure and mystical errors from SWIG:
```
/root/shogun/build/src/interfaces/python/shogunPYTHON_wrap.cxx: In function 'PyObject* _wrap_SomeObservedValue_put__SWIG_2(PyObject*, PyObject*)
':                                                                                                                                      
/root/shogun/build/src/interfaces/python/shogunPYTHON_wrap.cxx:837537:0: error: 'shogun_ObservedValue_put_scalar_dispatcher_Sl_int32_t_Sc_int32_
t_Sg___SWIG_2' was not declared in this scope                                                                                                   
         shogun_ObservedValue_put_scalar_dispatcher_Sl_int32_t_Sc_int32_t_Sg___SWIG_2((shogun::ObservedValue*)(arg1)->operator ->(),(std::string
 const &)*arg2,arg3);  
```
Any clue about what is going on? @karlnapf

Reply:
Don’t expose some to swig yet. We can do that soon but it’s not ready. You will need to use the old skool approach for now

Reply:
So just observed value raw pointer in the api